File: Data Structures & Algorithms/longest-consecutive-sequence/submission-2.java
class Solution {
    public int longestConsecutive(int[] nums) {

        if (nums.length == 0) return 0;
       Arrays.sort(nums);
       
       int min = nums[0];
        int count = 1;
        int maxCount = 1;
        for (int i = 1;i<nums.length;i++){
            if (nums[i] == min){
                continue;
            }
            else if(nums[i] == min + 1){
                count++;
                min = nums[i];
            }

            else{
                maxCount = Math.max(maxCount,count);
                count = 1;
                min = nums[i];
            }
        }

       return Math.max(count,maxCount);
    }
}

File: Data Structures & Algorithms/min-cost-climbing-stairs/submission-2.py
class Solution:
    def minCostClimbingStairs(self, cost: List[int]) -> int:
        n = len(cost)
        dp = [0] * n
        dp[0] = cost[0]
        dp[1] = cost[1]

        for i in range(2,len(cost)):
            dp[i] = min(dp[i-2],dp[i-1]) + cost[i]
        
        return min(dp[n-1],dp[n-2])